.github/workflows/weekly.yml

```diff
@@ -136,7 +137,8 @@ jobs:
     name: "Q 2.13 M 22.3 image"
     uses: graalvm/mandrel/.github/workflows/base.yml@default
     with:
-      quarkus-version: "2.13"
+      quarkus-version: "2.13.8.Final"
```

Comment:
Is it allowed to add a comment here. This is going to be hard to maintain unless one knows the details about why a branch doesn't work.

Reply:
Done

Reply:
Thanks, ship it!